DOCS/canonical-identity-platform-reference-kratos-observability.md

```diff
@@ -2,6 +2,8 @@ The Kratos charmed Operator of the IAM Platform provides integrations to the Can
 
 In this reference we will provide a technical description of the alert rules, and the dashboards the Kratos operator passes to the components of the COS. We will also showcase the Metrics we use to provide observability features.
 
+> See more: [metrics-endpoint and grafana dashboard integrations](https://charmhub.io/kratos/integrations)
```

Comment:
I think we can say it like on https://github.com/canonical/canonical-identity-platform-docs/blob/main/DOCS/canonical-identity-platform-how-to-integrate-with-oidc-compatible-charms.md?plain=1#L13
```suggestion
> See more: [Charmhub | Kratos > Integrations > `metrics-endpoint`,  `grafana-dashboard`](https://charmhub.io/kratos/integrations)
```